var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -139,6 +140,8 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
     variant("rythmos", default=False, description="Compile with Rythmos")
     variant("sacado", default=True, description="Compile with Sacado")
     variant("stk", default=False, description="Compile with STK")
+    variant("stk_unit_tests", default=False, description="Enable STK unit tests")
```

Comment:
Unit tests don't change the installed binaries so shouldn't be a variant. You should instead add `if self.run_tests and "+stk" in spec`

Reply:
I think I will just remove this for now.